pkg/apis/application/v1alpha1/types_test.go

```diff
@@ -4543,3 +4543,17 @@ func TestCluster_ParseProxyUrl(t *testing.T) {
 		}
 	}
 }
+
+func TestAppProject_ValidateSyncWindowDuplicates(t *testing.T) {
```

Comment:
Test should not be in the types_test.go file if the code changed is in app_project_types.go

Reply:
Is it okay then to move this to be in `TestAppProject_ValidateSyncWindowList` as you mentioned in the [comment](https://github.com/argoproj/argo-cd/pull/23636#discussion_r2180542523) below?